github/REVIEWING.md

```diff
@@ -0,0 +1,341 @@
+You can request changes as feedback text or as concrete content update suggestions, as shown in the [GitHub review guidelines](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) and add your request for changes.
+
+Locally you can get a unified view of the changes inside files.
+You can open a file and check out its entirety: existing content and newly contributed content.
+You can check the newly contributed content and make sure that:
+
+- It is consistent with existing content: same structure, same format, same conventions.
+- It is readable and integrates well with existing content.
+
+Finally, you want to be sure that the content is well aligned with the commits it is part of.
+Each commit should do one thing and do one thing well, and the content and commit should be aligned.
+
+For that, go through each commit in the contribution, from the oldest to the most recent one.
+Find the commit IDs by using:
+
+```console
+git log
+```
+
+Go to the history and get the oldest commit ID (the commit hash) and then check it.
+Then get the next commit ID and check it.
+And so on.
+
+To check each commit use:
+
+```console
+git show <commit_id>
+```
+
+where `<commit_id>` is the ID (hash) of the commit.
+
+When checking the commit be sure that:
+
+- The commit message and the commit contents correspond.
+  The commit message should not state anything more or less than the contents.
+
+- The commit is "atomic".
+  That is, it does one thing and one thing well.
+
+  You may have two situations:
+
+  1. The commit is too "large", it has multiple contributions.
+     In this case, the feedback should be to split to commit in smaller "atomic" commits, each per contribution.
+
+  1. The commit is too small.
+     There are multiple commits related to the same contribution.
+     In this case, the feedback should be to squash the commits into a single commit.
+
+In case of issues with individual commits, you will reference them by ID (or link) in the general review message.
+You can get a link to the commit in the `Commits` tab in the pull request interface on GitHub.
+
+## Test Changes
+
+In the above steps, you checked the contribution without actually using it.
+Changes must be tested to make sure they are correct and don't break anything.
+
+Ideally, the repository has CI/CD pipelines configured that automate testing of common items.
+This generally relies on the repository itself featuring tests that can be called automatically.
+
+Even if this is the case, there may be specific tests that you want do manually.
+
+When reviewing documentation contributions, make sure you run the documented actions (commands or otherwise) to ensure they work OK.
+
+Testing is generally particular to the repository.
+Do your best in ensuring that contributions work as expectedly and they don't introduce additional bugs.
```

Comment:
I would insist on testing the actual features the contribution adds, not just testing that nothing breaks. I feel like this is often missed both by the reviewers and by the automated tests.

Somthing like:
```
If the contributions add a new feature, make sure to test it properly, do not focus soley on regression testing.
Ask the author to provide instructions on how to test the new features, or, even better, to add test cases to the pull request.
```